### DISCUSSION TITLE
Have Pinned Tabs Reopen to Original Pin?

### DISCUSSION THREAD

**Discussion by Locke5368** [mbnuqw/sidebery#1674] [Q&A]
Hey everyone, tried searching for this and couldn't find it. I was wondering if in Sidebery there was a way to make it so that the pinned tabs reopen to their original pin on startup? Like if I pinned youtube.com watched a few videos, closed it, then reopened it, it would return to youtube.com.

Or if there was a way to pin bookmarks to the pinned tab area that would be pretty much the same, I think. Thanks for the help!

**Comment by pweiskircher** [comment]
I would also like it if I could just say "go back to original pin". Arc has that where you can just double click it (or something) and it goes back to the URL when you pinned it. Super nice.
